src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -28,30 +28,29 @@ export class CalcitePickList {
 
   /**
    * When true, the items will be sortable via drag and drop.
-   * Only applies when mode is configuration
+   * This only applies when mode is configuration.
    */
   @Prop({ reflect: true }) dragEnabled = false;
 
   /**
-   * Mode controls the presentation of the items in their selected and deselected states.
-   * Selection mode shows either radio buttons or checkboxes depending on the value of multiple
-   * Configuration mode relies on a color highlight on the edge of the item for selected
-   * Mode must be set to configuration for drag and drop behavior to work.
+   * This mode controls the presentation of the items in their selected and deselected states.
+   * Selection mode shows either radio buttons or checkboxes depending on the value of 'multiple'.
+   * Configuration mode relies on a color highlight on the edge of the item for the selected.
+   * For drag and drop behavior to work, configuration mode must be set.
```

Comment:
This should be marked deprecated. Can you double check if you merged in latest from master?